Directory.Build.props

```
@@ -1,11 +1,11 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
     <LangVersion>9.0</LangVersion>
-    <Nullable>enable</Nullable>
+    <Nullable>disable</Nullable>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);1591;1998;NU5105;CA1014</NoWarn>
+    <NoWarn>$(NoWarn);1591;1998;NU5105;CA1014;CA1014;CA2007;CA1031</NoWarn>
```

Nit: Duplicated rule
```suggestion
    <NoWarn>$(NoWarn);1591;1998;NU5105;CA1014;CA2007;CA1031</NoWarn>
```